README.md

```diff
@@ -342,6 +342,10 @@ Address Family Identifier (AFI). Required. Valid values are `ipv4` and `ipv6`.
 ##### `safi`
```

Comment:
:-1:   Let's hold off on this review until we can get a consensus on the node utils implementation.